app/assets/javascripts/application.js

```diff
@@ -1,3 +1,3 @@
 //= require jquery
 //= require jquery_ujs
-//= require_tree .
+//= require_tree .
```

Comment:
no newline at EOF! Consider adding this as a default to sublime (I think that's the editor you use?) http://www.sublimetext.com/forum/viewtopic.php?f=3&t=12118